hnn_core/gui/gui.py

```diff
@@ -28,79 +29,101 @@
                         VBox, link)
 from ipywidgets.embed import embed_minimal_html
 
+_spectrogram_color_maps = [
+    "jet",
```

Comment:
can we remove the "jet" colormap? It's not a good colormap for power calculation. Power is a positive quantity for which diverging colormap is not recommended.